src/static/js/home.js

```diff
@@ -0,0 +1,15 @@
+const menu = document.querySelector(".menu");
```

Comment:
Let's use single quotes for consistency throughout the JS
```suggestion
const menu = document.querySelector('.menu');
```